src/main/java/com/alipay/oceanbase/rpc/protocol/payload/impl/execute/aggregation/TableAggregationResult.java

```diff
@@ -0,0 +1,66 @@
+/*-
+ * #%L
+ * OBKV Table Client Framework
+ * %%
+ * Copyright (C) 2021 OceanBase
+ * %%
+ * OBKV Table Client Framework is licensed under Mulan PSL v2.
+ * You can use this software according to the terms and conditions of the Mulan PSL v2.
+ * You may obtain a copy of Mulan PSL v2 at:
+ *          http://license.coscl.org.cn/MulanPSL2
+ * THIS SOFTWARE IS PROVIDED ON AN "AS IS" BASIS, WITHOUT WARRANTIES OF ANY KIND,
+ * EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO NON-INFRINGEMENT,
+ * MERCHANTABILITY OR FIT FOR A PARTICULAR PURPOSE.
+ * See the Mulan PSL v2 for more details.
+ * #L%
+ */
+
+package com.alipay.oceanbase.rpc.protocol.payload.impl.execute.aggregation;
+
+import com.alipay.oceanbase.rpc.mutation.Row;
+import com.alipay.oceanbase.rpc.protocol.payload.impl.ObObj;
+import com.alipay.oceanbase.rpc.stream.QueryResultSet;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TableAggregationResult {
+    public TableAggregationResult(QueryResultSet queryResultSet_, Map<Integer, String> index_) {
+        this.queryResultSet = queryResultSet_;
+        this.index = index_;
+        this.isInit = false;
+    }
+    public void init() throws Exception {
```

Comment:
I guess init() could be done when the result starts to construct.

Reply:
fixed